src/StateService/StatePlugin.cs

```diff
@@ -176,36 +176,36 @@ private void OnVerifyProof(UInt256 root_hash, string proof)
         }
 
         [RpcMethod]
-        public JToken GetStateRoot(JArray _params)
+        public JToken GetStateRoot(JArray @params)
```

Comment:
same here and in the rest of them, I don't like the @

Reply:
Using @ for reserved words to use them as a variable is a C# standard. But I'll update it. And please, we are working on the core, not some personal project, please avoid using "like" or "not like", it sounds like you are using your personal feelings instead of professional judgements (though i know you are not).

Reply:
> Using @ for reserved words to use them as a variable is a C# standard. But I'll update it. And please, we are working on the core, not some personal project, please avoid using "like" or "not like", it sounds like you are using your personal feelings (though i know you are not) instead of professional judgements.

Could you show me where said that use a reserved word is a good practice?

Reply:
I am not saying using reserved word is a good practice, cause if i were the one who wrote the code, i will not use `params` as the parameter. But it is params and is used as _params, which is definately not a good practice. You said we should have a good code style, yet you allow someone to use `_params` as the name of the parameter.

Reply:
@Liaojinghui, you're trying to change some established code. `_params` was there since like forever, so for good or bad, but everyone just got used to it being this way. If all methods consistently use this name, no one can really object adding another one using the same style, that's just consistency. Is it a bad name? Sure it is. But just making it `@params` is not an improvement to me, at least not worth the trouble.

`parameters` is a different story, nice to have it instead and have it used consistently for every method as well. :+1: